### PR TITLE
commonlib: add vulnerability data

### PR DESCRIPTION
**Pull request by thc202**
Bundle vulnerability data in the add-on and provide it through the core classes.
Update add-ons to depend on the newer version of commonlib to force the data to be available.

Part of zaproxy/zaproxy#8012.

### CODE REVIEW

**Comment by thc202** [comment]
I didn't add the translations files, those will be added by zapbot when syncing from Crowdin.

File: addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/internal/vulns/LegacyVulnerabilities.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import java.lang.reflect.Proxy;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.parosproxy.paros.Constant;
+
+/** Loads and provides {@code Vulnerability} using (legacy) core classes. */
```
Comment (kingthorin) [inline addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/internal/vulns/LegacyVulnerabilities.java:33]: That functionality is going to ultimately be migrated here?

**Review by kingthorin — APPROVED** [review]

File: addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/internal/vulns/LegacyVulnerabilities.java (hunk 1 above)
Comment (thc202) [reply addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/internal/vulns/LegacyVulnerabilities.java:33]: Yes, to stop relying on core completely.

Comment (kingthorin) [reply addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/internal/vulns/LegacyVulnerabilities.java:33]: Perfecto!

**Comment by thc202** [comment]
Addressed a conflict and added a missing dependency version change.

**Review by kingthorin — APPROVED** [review]

**Comment by ricekot** [comment]
Has conflicts again...

**Comment by thc202** [comment]
Done.

**Pull request merged**